src/pages/workspace/WorkspaceProfileSharePage.tsx

```diff
@@ -53,6 +74,27 @@ function WorkspaceProfileSharePage({policy}: WithPolicyProps) {
                 />
                 <ScrollView style={[themeStyles.flex1, themeStyles.pt2]}>
                     <View style={[themeStyles.flex1, isSmallScreenWidth ? themeStyles.workspaceSectionMobile : themeStyles.workspaceSection]}>
+                        <View style={[themeStyles.mh5, themeStyles.mv1]}>
```

Comment:
In our Figma designs, we are using 12px of top padding to wrap the entire view that has the headline, explainer text, and then subsequent UI. 

For the headline itself, it has a margin below it of 8px and then the paragraph text starts. Can you clean up some of these styles so they match what we have in [Figma](https://www.figma.com/design/oFxo6zAu4HRJkpuzY18nlV/Workspace-Share?node-id=0-1&t=msI3usHkhTDAbDCV-1)? Thanks!